stdlib/socketserver.pyi

```diff
@@ -74,29 +75,29 @@ class TCPServer(BaseServer):
         allow_reuse_port: bool
```

Comment:
If we're updating the parameter annotation in `__init__`, we need to update the variable annotation as well

```suggestion
        allow_reuse_port: bool
   sever_address: _AfInetAddress
```